website/blog/2022-04-19-dbt-cloud-postman-collection.md

```diff
@@ -0,0 +1,184 @@
+>* check out the [collection documentation](https://documenter.getpostman.com/view/14183654/UVsSNiXC#auth-info-7ad22661-8807-406b-aec0-34a46b671aac) to learn how to use it.
+
+The dbt Cloud API has well-documented endpoints for creating, triggering and managing dbt Cloud jobs. But there are other endpoints that aren’t well documented yet, and they’re extremely useful for end-users. These endpoints exposed by the API enable organizations not only to orchestrate jobs, but to manage their dbt Cloud accounts programmatically. This creates some really interesting capabilities for organizations to scale their dbt Cloud implementations.
+
+The main goal of this article is to spread awareness of these endpoints as the docs are being built & show you how to use them. 
+
+<!--truncate-->
+
+You can use this blog post as an entry point to this [postman collection](https://documenter.getpostman.com/view/14183654/UVsSNiXC#auth-info-7ad22661-8807-406b-aec0-34a46b671aac) to help you automate previously manual tasks, such as managing dbt Cloud account infrastructure, creating dbt Cloud projects, database connections, and managing users.
+
+Please keep in mind that the collection is not an evergreen doc. We are actively developing and updating the API endpoints used to interact with dbt Cloud, so endpoint urls and / or expected request formats may change. Each endpoint in the collection provides comment capabilities, so please notify us there if something looks amiss.
+
+With all of that being said, why should dbt users care about automating dbt Cloud account management?
+
+## What problems does programmatic dbt Cloud account management solve?
+
+When first learning dbt Cloud and operating at a smaller scale, most end users tend to favor manual, GUI based workflows for creating and managing their account infrastructure. Many of our customers start off this way, but they usually end up running into these bottlenecks as the number of jobs and projects multiplies. Here are some examples of the kind of bottlenecks I hear our growing customers talk about.
+
+- “I find myself wasting time manually clicking through the UI to manage infrastructure, especially after deploying more than a handful of projects and environments.”
+- “I don’t have peace of mind that my production environment is configured exactly as intended. If a new team member deletes or changes something, I have to run a fire drill to fix it.”
+- “My organization requires me to run any environmental configuration changes through code review and version control.”
+
+We usually advise our customers to leverage API requests to automate these typically manual tasks, and you should, too!
+
+## Closing Thoughts
```

Comment:
I wonder if we can reword this from 'Closing Thoughts' to something else? Closing thoughts makes me think the content would be over, but it digs into an example after that. Or maybe moving the current 'Closing Thoughts' to the end? Also @johnblust can correct me on capitalization, but to be consistent with other headers, should maybe be 'Closing thoughts' with second word lowercased. Lmk what people think!

Reply:
Hmm that's a valid point. I'll see if I can come up with a better name